server/src/main/scala/org/http4s/server/middleware/AutoSlash.scala

```diff
@@ -16,12 +15,13 @@ object AutoSlash {
   def apply[F[_]](service: HttpService[F])(implicit F: Monad[F]): HttpService[F] = Service.lift {
     req =>
       service(req).flatMap {
-        case _: Pass[F] =>
+        case Pass() =>
```

Comment:
This means the same thing but avoids a check against an erased type, right?

Reply:
I think so yes, it's also to be consistent with other places in the code.